bigtable/client/instance_admin.h

```diff
@@ -55,10 +55,14 @@ class InstanceAdmin {
 
   /**
    * Return the list of instances in the project.
-   * @return
    */
   std::vector<google::bigtable::admin::v2::Instance> ListInstances();
 
+  /**
+   * Return the list of clusters in the project.
+   */
```

Comment:
Since you have created an example (thanks!) please make it visible from here:

```C++
/**
 * Return the list of clusters in the project.
 *
 * **Example**
 * @snippet bigtable_samples_instance_admin.cc list clusters
 */
```

That will embed the snippet in the documentation.